Source/WebCore/Modules/indexeddb/server/UniqueIDBDatabase.cpp

```diff
@@ -546,6 +546,9 @@ void UniqueIDBDatabase::didFireVersionChangeEvent(UniqueIDBDatabaseConnection& c
         }
     }
 
+    if (!m_currentOpenDBRequest)
```

Comment:
Should this be inside notifyCurrentRequestConnectionClosedOrFiredVersionChangeEvent() ? Because it is weird to null check something that is not used later in the function.

Reply:
Yes, it seems we can just move this to `notifyCurrentRequestConnectionClosedOrFiredVersionChangeEvent` and drop the `ASSERT(m_currentOpenDBRequest);` in it; then we could remove the null check in `UniqueIDBDatabase::connectionClosedFromClient` too.